src/main/java/org/opentripplanner/util/time/TimeUtils.java

```diff
@@ -116,6 +117,12 @@ public static String timeToStrCompact(int time) {
     return RelativeTime.ofSeconds(time).toCompactStr();
   }
 
+  public static String timeToStrCompactNoSeconds(int time) {
```

Comment:
Maybe we could have a test for this in the TimeUtilsTest

Reply:
Done.